### PR TITLE
ZK-4322: Listbox position is wrong after calling setSelectied in Safari

### PR DESCRIPTION
**Pull request by b952426**
ZK-4322: Listbox position is wrong after calling setSelectied in Safari

### CODE REVIEW

**Review by scribetw — CHANGES_REQUESTED** [review]
1. simplify case
2. remove the patch

File: zktest/src/archive/test2/B95-ZK-4322.zul [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			<custom-attributes org.zkoss.zul.listbox.rod="true" org.zkoss.zul.nativebar="false"/>
+			<listhead>
+				<listheader label="Model ROD True" sort="auto"/>
+			</listhead>
+		</listbox>
+	</hlayout>
+	<button label="Test Listitem.setSelected()">
+		<attribute name="onClick">
+			model1.addToSelection(model1.getElementAt(model1.getSize()-1));
+			model2.addToSelection(model2.getElementAt(model1.getSize()-1));
```
Comment (scribetw) [inline zktest/src/archive/test2/B95-ZK-4322.zul:45]: model2

File: zktest/src/archive/test2/B95-ZK-4322.zul [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	<button label="Test Listbox.setSelectedIndex()">
+		<attribute name="onClick">
+			model1.addToSelection(model1.getElementAt(0));
+			model2.addToSelection(model2.getElementAt(0));
+		</attribute>
+	</button>
+	<button label="Test Listbox.setSelectedItem()">
+		<attribute name="onClick">
+			model1.addToSelection(model1.getElementAt(model1.getSize()-1));
+			model2.addToSelection(model2.getElementAt(model1.getSize()-1));
```
Comment (scribetw) [inline zktest/src/archive/test2/B95-ZK-4322.zul:57]: model2

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by DevChu — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**